src/ui/external-calendar-sources-dialog.ts

```diff
@@ -0,0 +1,446 @@
+/**
+ * Dialog for managing external calendar sources configuration
+ */
+
+import { Logger } from '../core/logger';
+import type { ExternalCalendarSource } from '../types/external-calendar';
+
+export class ExternalCalendarSourcesDialog extends Dialog {
```

Comment:
Use DialogV2 or ApplicationV2 as appropriate. Those are the new standards to my knowledge.